src/frameworks/solid.ts

```diff
@@ -22,7 +22,7 @@ export const solidFramework: ReactiveFramework = {
       read: () => memo(),
     };
   },
-  effect: (fn) => createEffect(fn),
+  effect: (fn) => createRenderEffect(fn),
```

Comment:
Why this change?

Reply:
same reason as above; the new unit tests weren't passing without it.